client/src/js/apis/fetchData.js

```diff
@@ -0,0 +1,55 @@
+// client/src/js/apis/fetchData.js
+
+import fetch from 'cross-fetch';
+
+const roomUrl = 'http://localhost:3003/rooms/';
+
+const getRoomInfo = async (roomId, NumberReviewsPerPage) => {
```

Comment:
Is there a reason why NumberReviewsPerPage starts with uppercase?

Reply:
no, I messed up with the camelCase here